guide/src/generate_text_using_the_vertex_ai_gemini_api.md

```diff
@@ -0,0 +1,127 @@
+<!-- 
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Generate text using the Vertex AI Gemini API
+
+In this guide, you send a text prompt request, and then a multimodal prompt and
+image request to the Vertex AI Gemini API and view the responses.
+
+## Prerequisites
+
+Completing this guide requires you to set up a Google Cloud project and enable
+the Vertex AI API. We recommend you use the [Vertex AI Quickstart] to complete
+these steps.
+
+## Adding the Vertex AI client library as a dependency
+
+The Vertex AI client library includes many features. Compiling all of them is
+relatively slow. To speed up compilation, you can just enable the features you
+need:
+
+```toml
+{{#include ../samples/Cargo.toml:aiplatform}}
+```
+
+## Send a prompt to the Vertex AI Gemini API
+
+First, initialize the client using the default settings:
+
+```rust,ignore,noplayground
+{{#include ../samples/src/gemini.rs:text-prompt-client}}
+```
+
+We need to build the model name. For simplicity, this example receives the
+project id as an argument and uses a fixed location (`global`) and model id
```

Comment:
nit: I'd prefer "ID" over "id", here and below.

Reply:
Done.